src/cmd/run.go

```diff
@@ -265,18 +260,15 @@ func runCommand(container string,
 
 	logrus.Debugf("Container %s is initialized", container)
 
-	if _, err := isCommandPresent(container, command[0]); err != nil {
-		if fallbackToBash {
-			fmt.Fprintf(os.Stderr,
-				"Error: command %s not found in container %s\n",
-				command[0],
-				container)
-			fmt.Fprintf(os.Stderr, "Using /bin/bash instead.\n")
+	commandLauncher := "export SHELL=\"$(getent passwd \"$(whoami)\" | cut -d : -f 7)\"; "
```

Comment:
The `toolbox shell` idea described in https://github.com/containers/toolbox/issues/908 would be a better way to do this.

It would avoid all the gymnastics with shell snippets, and might make the fallback code faster.

Reply:
@yann-soubeyrand has a work in progress at https://github.com/containers/toolbox/pull/1037